Source/PeleLM.cpp

```diff
@@ -9477,8 +9441,7 @@ PeleLM::writePlotFile (const std::string& dir,
     MultiFab::Copy(plotMF,*volfrac,0,cnt,1,nGrow);
 
     // set covered values for ease of viewing
-    // EM_DEBUG problem here if eb.regular
-    //EB_set_covered(plotMF, 0.0);
+    EB_set_covered(plotMF, 0.0);
```

Comment:
I remember that this line was crashing if the simulation is running with EB but no geometries involved.

Reply:
I did do a test run with EB all_regular and it worked for me. But more testing couldn't hurt.